platform-validation/scripts/compatibility.sh

```diff
@@ -393,6 +393,39 @@ perform_capability_check_container(){
     wrap_pass "capability_check_container" "Pass"
 }
 
+check_shared_library_dependency(){
+    wrap_debug "Checking shared library dependency for aziot-edged and aziot-identityd"
+
+    # check dependencies 
+    ret=$(need_cmd ldconfig) && $(need_cmd grep)
+    if [ $? != 0 ]; then 
+        wrap_warning "Skipping check_shared_library_dependency" 
+        return
+    fi
+
+    # set the crucial libaries in the variable
+    # IOTEDGE_COMMON_SHARED_LIBRARIES is for both aziot-edged and aziot-identityd 
+    IOTEDGE_COMMON_LIBRARIES="libssl.so.1.1 libcrypto.so.1.1 libdl.so.2 librt.so.1 libpthread.so.0 libc.so.6 libm.so.6 ld-linux-x86-64.so.2 libgcc_s.so.1"
```

Comment:
noob question: Do these libraries have the same names and versions on all platforms/distros?

Reply:
Yes they are the same. I verified it.

Reply:
Is it true for different architectures as well? armv7 arm64?

Reply:
I missed the arm archs. I have verified and added as well.